utils/prometheus/src/networking.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019 Parity Technologies (UK) Ltd.
+// This file is part of Substrate.
+
+// Substrate is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Substrate is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
+
+use async_std::pin::Pin;
```

Comment:
Can this code be moved inside a helper library in prometheus?

Reply:
Are you talking about writing /test code?

Reply:
About `networking.rs`, the code seems pretty generic

Reply:
Oh, alright,
Should I write that code in /prometheus/src/lib.rs?

Reply:
or 
I can create PR to use networking.rs code to be put in rust-prometheus’s helper library if the helper library means the cargo package. There is other way to import networking.rs in grafana-datasource module. Which would you prefer?

Reply:
Exposing it in reusable way from grafana module and reusing here seems fine to me

Reply:
complete
Imported by changing networking.rs in the grafana module to pub mod.